src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,99 @@
+package duke;
+
+import java.util.Scanner;
+
+
+import java.util.ArrayList;
+import java.io.IOException;
+
+ 
+
+
+
+public class Duke {
+    public static void main(String[] args) throws IOException, ClassNotFoundException {
+
+
+     
+        //print welcome message, ask for user input
+        System.out.println("  ───── ･ ｡ﾟ☆: *.☽ .* :☆ﾟ. ───── \n Hello! I'm Broccoli the dinosaur \n           <|°▿▿▿▿°|/ \n      What can I do for you? \n   ──── ･ ｡ﾟ☆: *.☽ .* :☆ﾟ. ────" ); 
+        //getInput();
+        
+
+
+        ArrayList<Task> array=new ArrayList<Task>();
+        String divider = "    ═══*.·:·.☽✧    ✦    ✧☾.·:·.*═══";
+        
+        try {
+            SaveData.loadFile(array);
+        } catch(Exception e) {
+            System.out.println(e.getMessage());
+        }
```

Comment:
Should we be more specific about what exception to catch? I guess you want the following
```
} catch(IOException e) {
```
But I am not very sure about the exact name of the exception that `loadFile` could throw, so please feel free to correct me if I am wrong. : )